src/test/java/uk/ac/ebi/atlas/configuration/AbstractContainerBaseTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.ebi.atlas.configuration;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import org.flywaydb.core.Flyway;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import javax.sql.DataSource;
+
+abstract class AbstractContainerBaseTest {
```

Comment:
Why is this class abstract?
Why does it have `Abstract` in the name when it’s abstract already?
Why do we need this class?

Reply:
We don't need this class. I tried to implement it in the TestConfig class, but I got some issues. So ended up this class.